lib/engine/company.rb

```diff
@@ -47,7 +47,7 @@ def close!
       all_abilities.dup.each { |a| remove_ability(a) }
       return unless owner
 
-      owner.companies.delete(self)
+      owner.companies.delete(self) if owner.player? || owner.corporation?
```

Comment:
why was this added? will thins break other games?

Reply:
Because I need to close a company even if it's owned by the bank. The bank doesn't have a companies method, so the original code will blow up.  This shouldn't break anything since the only things that have companies are players and corporations.

I could write it like this:
`owner.companies.delete(self) if owner.respond_to?(:companies)`
Would that be preferable?